Task/Runner/Nuke/Build.cs

```diff
@@ -61,7 +61,7 @@ public class Build : NukeBuild
         .Description("Verify Nuke build targets for documentation and naming conventions")
         .Executes(() =>
         {
-            var nukeSolution = RootDirectory / "Ops" / "TaskRunner" / "Nuke.sln";
+            var nukeSolution = RootDirectory / "Task" / "Runner" / "Nuke.sln";
```

Comment:
There are three occurrences of `RootDirectory / "Task" / "Runner"` in this file. Can this be extracted to its own variable to make the code more DRY?